app/src/components/inputs/inputConditionalAttributes/inputConditionalAttributes.jsx

```diff
@@ -105,11 +150,31 @@ export class InputConditionalAttributes extends Component {
     }
   };
 
+  parseTagsToString = (attributes) => {
+    return attributes.map((attr) => `${attr.key}:${attr.value}`).join(',');
```

Comment:
What if the attribute will contain only value or only key? Will it return smth like this `build:1.2.3,:attrValue`?